### ISSUE TITLE
Can openemu be uninstalled while maintaining save states?

### ISSUE THREAD

**Issue by shinsukato** [OpenEmu/OpenEmu#3414]
I've been having lots of freezing issues with OpenEmu recently after doing some playing around with my environment.

Can I safely remove and re-install OpenEmu without deleting my save states?

**Comment by clobber** [comment]
This is the [XY Problem](http://xyproblem.info/). Instead you should tell me which version of macOS and OpenEmu you are using, and which system and game(s) you are experiencing the freeze in. Re-copying the app to your Applications folder or deleting some files won't solve it.

**Comment by shinsukato** [comment]
Sure.

OpenEmu v2.0.5 719-g369c3a2-Release

System info:
MacOS 10.13.1
Model Name:	MacBook Pro
Model Identifier:	MacBookPro14,1
Processor Name:	Intel Core i5
Processor Speed:	2.3 GHz
Number of Processors:	1
Total Number of Cores:	2
L2 Cache (per Core):	256 KB
L3 Cache:	4 MB
Memory:	8 GB

Games encountered in:
PS1 Final Fantasy 9 USA (v1.1)
SNES Secret of Mana USA (unpatched)
NES Final Fantasy 2 

Repro Steps:
1) Load the game on question.
2) Connect a Bluetooth DS4 controller.
3) Play for around 1.5 hours, making liberal use of the fast forward function.

Issue:
The game froze, and the entire application seemed to crash. UI elements respond, I can get into application preferences, but I can't end the emulation of the game. I can't load a previous save state or restart the console. If I try to quit OpenEmu I get the usual warning dialogue, but the application does not close. I have to Force Quit. When I open OpenEmu again and try to load the game, I am returned to my previous 'on-quit' save state, but any save states I created subsequently are still available.

On a side-note, where are OpenEmu save states located and how would I preserve them if I removed it and re-installed it, just in case this issue was caused by some tinkering I was doing?
